packages/vm/src/wasm_backend/compile.rs

```diff
@@ -22,8 +22,10 @@ mod tests {
     static CONTRACT: &[u8] = include_bytes!("../../testdata/floaty.wasm");
 
     #[test]
-    #[should_panic]
     fn contract_with_floats_fails_check() {
-        compile(CONTRACT, None).unwrap();
+        let err = compile(CONTRACT, None).unwrap_err();
+        assert!(err
+            .to_string()
+            .contains("Non-deterministic operator detected"));
```

Comment:
This is little more explicit / informative, but previous version should work. I mean, should panic as expected.

Reply:
Those are two independent changes in one: 1. make test work by ensuring there are float operations in the Wasm and 2. refactor test to nicely test the error instead of converting the error to panic.

Reply:
Yeah, the updated `.wasm` file is the thing that actually fixes the test (even in its previous version). I think I previously updated the wrong one by accident. Sorry for the confusing commit.